storage/store.go

```diff
@@ -118,7 +120,27 @@ func (a *awsBackend) WatchList(ctx context.Context, key string, opts storage.Lis
 // The returned contents may be delayed, but it is guaranteed that they will
 // match 'opts.ResourceVersion' according 'opts.ResourceVersionMatch'.
 func (a *awsBackend) Get(ctx context.Context, key string, opts storage.GetOptions, objPtr runtime.Object) error {
-	panic("not implemented") // TODO: Implement
+	listOut, err := a.s3.ListObjectVersions(ctx, &s3.ListObjectVersionsInput{
+		Bucket: aws.String(a.bucket),
+		Prefix: &key,
+	})
+	latestCommitted, err := getLatestCommittedVersion(ctx, a.s3, a.bucket, key, listOut)
+	if err != nil {
+		return storage.NewKeyNotFoundError(key, 0)
+	}
+	data, err := a.s3.GetObject(ctx, &s3.GetObjectInput{
+		Bucket:    aws.String(a.bucket),
+		Key:       aws.String(key),
+		VersionId: latestCommitted.VersionId,
+	})
+	if err != nil {
+		return err
+	}
+	raw, err := ioutil.ReadAll(data.Body)
+	if err != nil {
+		return err
+	}
+	return decode(a.codec, a.Versioner(), raw, objPtr, 1)
```

Comment:
revision should not be hardcoded as `1`. Need to retrieve if from tags.

Reply:
changed to get from tagset